README.md

```diff
@@ -15,8 +15,19 @@
     </a>
 </p>
 
-`connqc` is a tool to control the quality of the network transport between multiple regions.
-It consists of a server that listens for TCP probing messages, and a client that sends said messages.
+`connqc` is a tool to check the quality of the network transport between networked hosts.
+We have, in our past encountered the following problems in the wild between two hosts:
+- The network connection between two hosts is not stable.
+- The network connection between two hosts may seem stable, but the firewall in between may be dropping packets.
+- The network connection between two hosts may seem stable, but the firewall in between may be dropping packets after a certain amount of time.
+- The network connection between two hosts may seem stable, but the firewall in between may be dropping packets after a certain amount of time, but only if the connection is established for a while.
+- The network connection between two hosts may seem stable, but certain ports or port combinations are blocked.
+- The network connection seems stable, but the routes taken are flipping and thus creating spontaneous connection issues.
+- Some other weird stuff.
+
+As most of you who may have found themselves in this situation, we have fiddled with ping, netcat, iperf, and other tools, but found that most of these would not establish a connection, keep it up without user interaction and report on the quality of the connection. This is why we developed this tool, to help others in the same situation.
```

Comment:
This sentence is a little weird, perhaps:
```suggestion
While other tools exist that are similar to `connqc`, such as **ping**, **netcat** and **iperf**, they do not establish a connection that is maintained without user interaction while also reporting the connection quaily. For this reason, `connqc` was developed.
```

Reply:
@nrwiersma I agree that the sentence is strange, and I prefer your suggestion, but I'd even suggest removing the last sentence because I feel like we don't need to explicitly explain that this is why we developed the tool, as it is implicitly a given based on the previous answer. And the reason for developing the tool, ultimately, does not matter IMO. What should be clear is what it does, and optionally how it compares to other solutions, but not so much why it exists.

Reply:
```suggestion
While other tools exist that are similar to `connqc`, such as **ping**, **netcat** and **iperf**, they do not establish a connection that is maintained without user interaction while also reporting the connection quality.
```

* Removes last sentence
* Fixes typo to `quality`